internal/gps/result_test.go

```diff
@@ -35,12 +34,6 @@ func init() {
 			}, nil),
 		},
 	}
-
-	// just in case something needs punishing, kubernetes is happy to oblige
-	kub = atom{
-		id: pi("github.com/kubernetes/kubernetes"),
-		v:  NewVersion("1.0.0").Is(Revision("528f879e7d3790ea4287687ef0ab3f2a01cc2718")),
```

Comment:
Let's leave this in; as the comment indicates, it's there mostly to do the work of picking out a known kubernetes rev to work from. Even though it's not currently used anywhere, simply making that choice has some potential value down the road.